### PR TITLE
feat: add number to Title

### PR DESCRIPTION
**Pull request by nata-kab**
## Jira issue

JIRA issue: [FSU-73](https://klau.atlassian.net/browse/FSU-73)

## Describe your changes

Added the ability to add a number to the title. 

I added a new Count component because we will use numbers in the same style but different sizes not only in "Title component" but also in other places on the page.

## Checklist before requesting a review

- [x] Checked which branch you would merge.
- [x] Checked if the code follows the style guidelines of this project (use eslint and prettier).
- [x] Performed a self-review of code (bugs, clean code rules, etc).
- [x] Checked if added variables have unique names.
- [x] Checked if added code/function is not a duplicate. 
- [x] Tests have been added if possible.
- [x] New and existing unit tests pass locally with my changes.

### CODE REVIEW

File: packages/design-system/components/title/Title.jsx [hunk 1]
```diff
@@ -24,6 +26,7 @@ Title.propTypes = {
   badge: string,
   iconStart: string,
   iconEnd: string,
+  count: oneOfType([string, number]),
```
Comment (im-klau) [inline packages/design-system/components/title/Title.jsx:29]: Having this prop as a number and a string at the same time can be confusing. I'd think about keeping it simple and sticking to numbers only.

**Review by im-klau — COMMENTED** [review]
Actually, you've updated the existing component and created a new one (Count) from scratch and yet, the Pull Request is super easy to review and all the changes are well thought out 👏 

Well done! ❤️

File: packages/design-system/components/title/Title.jsx (hunk 1 above)
Comment (nata-kab) [reply packages/design-system/components/title/Title.jsx]: Done 🚀

**Review by im-klau — APPROVED** [review]

**Review by paulgrym — APPROVED** [review]
Everything looks great! You even remembered to update font import from google fonts ❤️

**Pull request merged**